components/actionbutton/stories/actionbutton.test.js

```diff
@@ -17,12 +17,14 @@ export const ActionButtons = (args, context) => {
 			${Template({
 				...args,
 				hasPopup: "true",
+				hasLongPress: true,
```

Comment:
Do we need an `isOpen` arg in any of the tests? Not sure if it's valuable, especially if other args are already covering the style changes.